util.sh

```diff
@@ -319,6 +311,21 @@ function aws_secret {
         echo -e "\t* Rerun 'aws-mfa' to ensure MFA session token is up-to-date" > /dev/stderr
         exit 1
     fi
+}
+
+# get the content of the aws secret subkey
+# usage:
+# aws_secret <secretid> <key>
+function aws_secret {
+    if ! which jq &> /dev/null; then
+        echo -e ${RED}jq is missing${END_COLOR} > /dev/stderr
+        if [ $(uname -s) = "Darwin" ]; then
```

Comment:
we should probably give folks an installer for all required utilities. This is fine for now as we prepare for the demo but would be against requiring things like jq to be present for things to work. The fact that we need these utils perhaps means that `util.sh` as a standalone place for handling those pieces is not ideal (more generally - bash is not very readable once it goes beyond basic stuff and is more error prone compared to say a set of go/ python utils)

Reply:
the benefit of bash is that its usually pre-installed so its easy to integrate with the repos without requiring too many tools but in the long run I can see splitting more advanced functionality into separate internal tools